articles/azure-functions/functions-bindings-sendgrid.md

```diff
@@ -348,9 +345,9 @@ The following table lists the binding configuration properties available in the
 
 | *function.json* property | Attribute/annotation property | Description | Optional |
 |--------------------------|-------------------------------|-------------|----------|
-| type || Must be set to `sendGrid`.| No |
-| direction || Must be set to `out`.| No |
-| name || The variable name used in function code for the request or request body. This value is `$return` when there is only one return value. | No |
+| type |n/a| Must be set to `sendGrid`.| No |
+| direction |n/a| Must be set to `out`.| No |
+| name |n/a| The variable name used in function code for the request or request body. This value is `$return` when there is only one return value. | No |
```

Comment:
Adding "n/a" to be consistent with the way the Configuration tables are in the other bindings